database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -17,6 +17,7 @@ public function up()
             $table->string('name');
             $table->string('email')->unique();
             $table->string('password');
+            $table->integer('type');
```

Comment:
The role thing isn't that typical and just adding more redundant tables in my opinion. same case as the status table, better leaving it as an attribute to avoid that